app/src/main/java/org/metabrainz/mobile/presentation/UserPreferences.java

```diff
@@ -19,9 +19,10 @@
     private static final String PREFERENCE_RATINGS_TAGS = "ratings_tags";
     private static final String PREFERENCE_SYSTEM_LANGUAGE = "use_english";
     private static final String PREFERENCE_ONBOARDING = "show_onboarding";
+    private static final String SHARED_PREF_FILE = "shared_pref_file";
 
     private static SharedPreferences getPreferences() {
-        return PreferenceManager.getDefaultSharedPreferences(App.getContext());
+        return App.getContext().getSharedPreferences(SHARED_PREF_FILE, Context.MODE_PRIVATE);
```

Comment:
The [PreferenceManagerCompat](https://developer.android.com/reference/androidx/preference/PreferenceFragmentCompat) documentation mentions:
>To retrieve an instance of SharedPreferences that the preference hierarchy in this fragment will use by default, call PreferenceManager.getDefaultSharedPreferences(android.content.Context) with a context in the same package as this fragment.

Hence, replacing the `getDefaultSharedPreferences` would also need changing the `SettingsFragment` to use it.

Another solution is to use the AndroidX Preference Library as this [deprecation note](https://developer.android.com/reference/android/preference/PreferenceManager) mentions.

Reply:
Thanks a lot for this suggestion! I will work on it straightaway! :)